cmd/centaurid/cmd/bech32_convert.go

```diff
@@ -16,9 +16,9 @@ func AddBech32ConvertCommand() *cobra.Command {
 		Long: `Convert any bech32 string to the centauri prefix
 
 Example:
-	centaurid debug bech32-convert akash1a6zlyvpnksx8wr6wz8wemur2xe8zyh0ytz6d88
```

Comment:
tbh may be just delete this command.

there is https://github.com/strangelove-ventures/bech32cli can go get run or nix run, whatever. no need for generic tool here.

Reply:
but retaining code is also ok. just less code less work.

Reply:
method to do it without using a cli https://cosmosdrops.io/en/tools/bech32-converter